resources/cloudwatch-alarms.go

```diff
@@ -0,0 +1,60 @@
+package resources
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudwatch"
+)
+
+type CloudWatchAlarm struct {
+	svc       *cloudwatch.CloudWatch
+	alarmName *string
+}
+
+func init() {
+	register("CloudWatchAlarm", ListCloudWatchAlarms)
+}
+
+func ListCloudWatchAlarms(sess *session.Session) ([]Resource, error) {
+	svc := cloudwatch.New(sess)
+	resources := []Resource{}
+
+	params := &cloudwatch.DescribeAlarmsInput{
+		MaxRecords: aws.Int64(100),
+	}
+
+	for {
+		output, err := svc.DescribeAlarms(params)
+		if err != nil {
+			return nil, err
+		}
+
+		for _, metricAlarm := range output.MetricAlarms {
+			resources = append(resources, &CloudWatchAlarm{
+				svc:       svc,
+				alarmName: metricAlarm.AlarmName,
+			})
+		}
+
+		if output.NextToken == nil {
+			break
+		}
+
+		params.NextToken = output.NextToken
+	}
+
+	return resources, nil
+}
+
+func (f *CloudWatchAlarm) Remove() error {
+
+	_, err := f.svc.DeleteAlarms(&cloudwatch.DeleteAlarmsInput{
+		AlarmNames: []*string{f.alarmName},
```

Comment:
I think this is good. The aws utils seem to be only for easy casting.

Reply:
Actually, this is not a cast. It is just putting a single item into a slice which is perfectly fine.

Reply:
OK, thanks! Glad to know I'm not too bad a grade school Go!